Makefile

```diff
@@ -49,7 +49,7 @@ endif
 .SECONDARY: $(call to-hash,$(ALLS))
 
 .update-pip:
-	@pip install -U 'pip'
+	@python -m pip install --upgrade pip
```

Comment:
Why is this any different?

Reply:
Ah, I see what you mean. The log is here: https://github.com/aio-libs/aiohttp/pull/5647/checks?check_run_id=2435935600#step:6:14